### PR TITLE
[CXF-8724]:Add codegen module back

### PR DESCRIPTION
**Pull request by jimma**

### CODE REVIEW

File: systests/wsdl_maven/codegen/src/it/cxf-4004/pom.xml [hunk 1]
```diff
@@ -37,12 +37,12 @@
                         <artifactId>cxf-xjc-ts</artifactId>
                         <version>${cxf.xjc-utils.version}</version>
                     </dependency>
-                    <dependency>
+                    <!--dependency-->
```
Comment (reta) [inline systests/wsdl_maven/codegen/src/it/cxf-4004/pom.xml:40]: thanks @jimma, LGTM, could you please add reference to https://issues.apache.org/jira/browse/CXF-8774 here, once the replacement is available, will be easy to trace the changes back.

Comment (jimma) [reply systests/wsdl_maven/codegen/src/it/cxf-4004/pom.xml:40]: @reta  Do you mean change the jira id in the commit message ? Or add some comment to https://issues.apache.org/jira/browse/CXF-8774  ?

Comment (reta) [reply systests/wsdl_maven/codegen/src/it/cxf-4004/pom.xml:40]: @jimma just a comment, thanks, so we know why it is commented out

Comment (jimma) [reply systests/wsdl_maven/codegen/src/it/cxf-4004/pom.xml:40]: Done. Thanks for review @reta

**Review by reta — APPROVED** [review]

**Pull request merged**